src/content/kitchen-sink.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Introduction to New Relic Android app
+contentType: page
+template: basicDoc
+topics:
+  - Mobile apps
+  - New Relic Mobile apps
+  - Android app
+japaneseVersion: ''
+---
+
+<Collapser>
```

Comment:
Taking a look at how this component is put together, it looks like each `CollapserRow` is also wrapped by its own `Collapser` component. Because of that, does it even make sense to wrap these rows from a user perspective?

I could however see this being useful for making sure a "collapser group" has the proper spacing in between each collapser row. What if we rename these slightly?

`Collapser` -> `CollapserGroup`
`CollapserRow` -> `Collapser`

The thinking is this: I should be able to use a single `Collapser` on its own. If I only have a single collapser, I shouldn't need to embed it in a `CollapserGroup` (hence the name) because there are no other collapser's I'd need to add space between. If I want multiple `Collapser`s next to each other (like this example), then I'd wrap it in a `Collapser` group to make sure there is proper space between them.

Thoughts on this idea?

Reply:
my thinking was eventually in the `<Collapser />` component we may add things like a button to open all `<CollapserRow>` rows on click. buuut that and maybe some styling on the container were the only use cases i could think of, really.

Reply:
Thinking on this more, I think I get what you're puttin' down. Will work on that today, may ping you for some input :)

Reply:
I probably should have provided an example to better explain this. Ideally I'd be able to use the collapser 1 of 2 ways:

```js
// on its own
<Collapser title="My title">
  Stuff
</Collapser>

// When I have multiple
<CollapserGroup>
  <Collapser title="My title">
    Stuff
  </Collapser>
  <Collapser title="My title 2">
    Stuff 2
  </Collapser>
</CollapserGroup>
```

`CollapserGroup` would be responsible here for adding the spacing between each of the `Collapser`s. Does that help clarify?

Reply:
> we may add things like a button to open all <CollapserRow> rows on click

That should still be possible with this design if we put it in the `CollapserGroup`. It may not make sense to show a "open all" button anyways if there is a single collapser.